temporalcli/commandsmd/commands.md

```diff
@@ -635,6 +635,29 @@ temporal workflow execute
 Includes options set for [workflow start](#options-set-for-workflow-start).
 Includes options set for [payload input](#options-set-for-payload-input).
 
+### temporal workflow history: Interact directly with a workflow's event history.
+
+Workflow history commands use this syntax: `temporal workflow history COMMAND [ARGS]`.
+
+### temporal workflow history convert: Converts an event history between formats.
```

Comment:
I wonder if this should be a command called `convert-history` instead of a new subcommand type

Reply:
Is there an existing command for extracting the history JSON using the CLI?  Because I couldn't find one and had to use the dev server web UI.  If we don't, I think we should add it as `temporal workflow history get`.  If we do, then I think you're probably right, but maybe we should make the history JSON extraction easier to find.

(`temporal workflow show -o json` *almost* does this, but it gives the decoded payloads, not the raw payloads, and `HistoryFromJSON` can't read JSON with decoded payloads.)

Reply:
> Is there an existing command for extracting the history JSON using the CLI?

Yes, though poorly named `workflow show`

> temporal workflow show -o json almost does this, but it gives the decoded payloads, not the raw payloads, and HistoryFromJSON can't read JSON with decoded payloads.

Not following exactly, what do you mean decoded payloads in this case? To you mean applying a payload codec?

Reply:
Oh, I think I found it, we should fix this for `workflow show` specifically. Today you need to call `temporal workflow show -o json --no-json-shorthand-payloads`.

Reply:
Ah, okay.  I went with `temporal workflow fix-history-json`.